_unittest/test_00_EDB.py

```diff
@@ -2146,12 +2146,14 @@ def test_129_hfss_simulation_setup(self):
 
         via_settings = setup1.via_settings
         via_settings.via_density = 1
+        via_settings.via_mesh_plating = True
         via_settings.via_material = "pec"
         via_settings.via_num_sides = 8
         via_settings.via_style = "kNum25DViaStyle"
 
         via_settings = edbapp.setups["setup1"].via_settings
         assert via_settings.via_density == 1
+        assert not via_settings.via_mesh_plating
```

Comment:
here I would add an if 2024.1 then assert is True right? then we do not need to modify the unitest once we migrate to the new version. The version is already there, because it is imported: from _unittest.conftest import desktop_version